types/jwt-decode/jwt-decode-tests.ts

```diff
@@ -1,7 +1,6 @@
- 
-import jwtDecode = require('jwt-decode');
- 
-let token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmb28iOiJiYXIiLCJleHAiOjEzOTMyODY4OTMsImlhdCI6MTM5MzI2ODg5M30.4-iaDojEVl0pJQMjrbM1EzUIfAZgsbK_kgnVyVxFSVo";
+import * as jwtDecode from "jwt-decode";
```

Comment:
It's the same. Read the section "I'm Sad About This; Rules are Dumb." about "hack" in your explanation.